network/src/main/java/javasabr/mqtt/network/user/ConfigurableNetworkMqttUser.java

```diff
@@ -0,0 +1,20 @@
+package javasabr.mqtt.network.user;
+
+import javasabr.mqtt.network.MqttConnection;
+import javasabr.mqtt.network.MqttNetworkSession;
+import javasabr.mqtt.network.message.out.ConnectAckMqtt311OutMessage;
+import org.jspecify.annotations.Nullable;
+import reactor.core.publisher.Mono;
+
+public interface ConfigurableNetworkMqttUser extends NetworkMqttUser {
+
+  MqttConnection connection();
+
+  void clientId(String clientId);
+
+  void session(@Nullable MqttNetworkSession session);
+
+  void reject(ConnectAckMqtt311OutMessage connectAsk);
```

Comment:
Does it support MQTT 5 message ?

Reply:
Mqtt5 version extends it so you can use with both